lighthouse-core/report/v2/renderer/report-renderer.js

```diff
@@ -138,6 +138,12 @@ class ReportRenderer {
       categories.appendChild(this._categoryRenderer.render(category, report.reportGroups));
     }
 
+    // Render a warning that perf numbers may be off when using headless chrome.
+    if (/HeadlessChrome/.test(report.userAgent)) {
+      const warning = this._dom.cloneTemplate('#tmpl-lh-headless-warning', this._templateContext);
+      reportSection.insertBefore(warning, categories);
```

Comment:
can this move above `categories` (and use `appendChild`) so that it's in the code in the same order it will appear in the report?

Reply:
can we add a warning-container to the report DOM permanently, and then this can easily appendChild reliably?